migrations/2023-11-03-114049_optimize-send-activity-size/up.sql

```diff
@@ -0,0 +1,31 @@
+-- Change the received_activity table to use less space. Instead of storing the full ap_id url,
+-- only store first 32 chars of hash which is enough to catch duplicates. This reduces the size
+-- of that column from from ~70 bytes to 33 bytes. Also drop id column which is unnecessary.
+-- The `published` time is stored by postgres as unix timestamp, so it only takes 8 bytes already.
+ALTER TABLE received_activity
+    ADD COLUMN ap_id_hash varchar(32);
```

Comment:
This should actually be a fixed size byte array instead of hex string to use less space.